components/miller/stories/miller.stories.js

```diff
@@ -80,9 +80,15 @@ export default {
 	},
 };
 
+/**
+ * Miller columns that allow both files and folders to be selected.
+ */
 export const BranchesSelectable = MillerGroup.bind({});
 BranchesSelectable.args = {};
 
+/**
+ * Miller columns that only allow files to be selected.
+ */
 export const FilesSelectable = Template.bind({});
 FilesSelectable.tags = ["!dev"];
```

Comment:
Do we want this in the sidebar at all? I don't think there's a way to view this story unless it's in the testing preview right now. Should I remove this tag?

Reply:
We can put it in the side navigation if it's hard to recreate the example on the default demo view.